### DISCUSSION TITLE
Dashboard Update Information for Clients and Sites

### DISCUSSION THREAD

**Discussion by Honoo92** [amidaware/tacticalrmm#1989] [Ideas]
Hi,

I encountered your project a few days ago, and I can only say that you are doing an amazing job!

I checked out some similar options, and your solution seems to be highly performant with a clear and attractive overview.

I would love to roll it out to all our customers, but I'm missing some crucial parts.
Maybe I haven't configured the frontend enough, but I created some custom fields specifically for clients and sites.

As I understand it, these custom fields are currently only used for scripts and reports.
The custom fields for each client are shown in the summary, which is a decent location for them.

However, what I need is for this information to be displayed directly on the dashboard—for example, the custom field "Address" for the site, which represents one of our customer locations.
There are a few such pieces of information that I would love to set and see while I'm in the dashboard.

I know this information is more suited for CRM than RMM, but since the options for custom fields on clients and sites are already available, I'm sure this would help a lot of people get more information in one overview.

Thanks and have a great Time. 
Im sure this project offers a great help and is getting bether in any update.

**Comment by dinger1986** [comment]
why not fill the site name in as the address? how would you envision this being show? Maybe do a rough mockup would help and open a feature request

**Reply by Honoo92** [reply]
Thanks for the fast reply.
It would be great to be able to treat the Sites and Clients as own entities (since they can get theire own custom fields).
The address was just a example they could have few more information, like for the clients: Manager, Customer Status, signed contracts and so on. (as custom fields there are no limits there, thats what i like the most!)

It could be shown exactly as the agents do allready, in the summary tab.
Right now when i select a client or site nothing is shown there just the information "No agent selected".
Maybe you could make a difference between 
selecting a client -> client summary 
selecting a site - > site summary
selecting one ore more agents -> agent summary

thanks and im looking forward how you think about this idea.

**Reply by silversword411** [reply]
Sounds like you want https://github.com/amidaware/tacticalrmm/issues/553

But in the mean time you could probably create a report that you can customize for client and site with summary data of whatever you want.
